ControllerKey.js

```diff
@@ -141,18 +144,18 @@ export class ControllerKey {
    * sign or verify data.
    *
    * If this ControllerKey is a controller of the HMAC, then the API for it can
-   * returned by passing only the key description. Otherwise, an OCAP-LD
-   * authorization capability must also be passed; without this capability,
-   * calls to the returned API will not be authorized to perform HMAC
-   * operations.
+   * be returned by passing only the key id and type. Otherwise,
+   * an OCAP-LD authorization capability must also be passed;
+   * without this capability, calls to the returned API will not
+   * be authorized to perform HMAC operations.
```

Comment:
Same thing with clearly sub-optimal spacing, sorry to be a drag on this.

Reply:
Addressed in https://github.com/digitalbazaar/web-kms-client/pull/2/commits/09d4ec93574132f2a7a5205559256f63a81ca903